app.py

```diff
@@ -104,6 +105,7 @@ def solver(os_name: str, os_version: str, python_version: str) -> None:
     """Purge solver data."""
     graph = GraphDatabase()
     graph.connect()
+    graph.purge_solver_documents(os_name=os_name, os_version=os_version, python_version=python_version)
     result = graph.delete_solved(os_name=os_name, os_version=os_version, python_version=python_version)
```

Comment:
If I read this correctly, this line can be dropped completely as the logic behind `purge_solver_documents` handles deletion:

```suggestion
```

Reply:
Correct, I will remove this line